app/views/step_by_step_pages/show.html.erb

```diff
@@ -21,12 +21,17 @@
     <% unless @step_by_step_page.can_be_published? %>
       <%= render "govuk_publishing_components/components/inset_text" do %>
         <h2 class="govuk-heading-m">To publish this step by step you need to</h2>
+
         <% if !@step_by_step_page.steps.any? %>
           <%= link_to('Add at least one step', new_step_by_step_page_step_path(@step_by_step_page), class: 'govuk-link govuk-link--no-visited-state') %>
-        <% elsif !@step_by_step_page.steps_have_content? %>
-          <%= link_to('Add content to all your steps', '#steps', class: 'govuk-link govuk-link--no-visited-state') %>
-        <% elsif !@step_by_step_page.links_checked_since_last_update? %>
-          <%= link_to "Check for broken links", { action: "check_links", step_by_step_page_id: @step_by_step_page.id}, method: :post, class: 'govuk-link govuk-link--no-visited-state' %>
+        <% else %>
+          <% unless @step_by_step_page.steps_have_content? %>
```

Comment:
Don't use `else` and `unless` together as it's very hard to read. I think it would be better to have an elsif:

```
if @step_by_step_page.steps.empty?
elsif !@step_by_step_page.steps_have_content?
```

Reply:
The structure is roughly:

```
if ...empty?
  link 1
else
        if !steps_have_content?
          link 2
        elsif some_other_condition?
          link 3
end
```

To allow for link 1 and link 2/3 to be displayed at the same time, this needs to be nested as above. But can do `if !` 👍

Reply:
I think it might be nicer to use a list of if statements. Once for each action.
e.g.
`if empty?`
`if no content?`
`if links not checked` 
each if could contain all of the logic it needs, e.g. should_check_links? could check whether there are any steps, and if the steps have content. This keeps the nested logic out of the view.